test/finance/test_readme_sample.py

```diff
@@ -17,13 +17,22 @@
 """
 
 import unittest
+import warnings
 
 from test.finance import QiskitFinanceTestCase
 
 
 class TestReadmeSample(QiskitFinanceTestCase):
     """Test sample code from readme"""
 
+    def setUp(self):
```

Comment:
Similar comment to above - if our readme sample is emitting deprecation warnings we should be fixing the sample - unless its from some underlying library that we have no control over.